src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/BrevModul/Brevskjema.tsx

```diff
@@ -351,6 +363,72 @@ const Brevskjema = ({ onSubmitSuccess, bruker }: IProps) => {
                         )}
                     </FritekstWrapper>
                 )}
+                {skjema.felter.fritekstAvsnitt.erSynlig && (
+                    <FritekstWrapper>
+                        <Label htmlFor={fritekstSkjemaGruppeId}>Legg til fritekst avsnitt</Label>
+                        {visFritekstAvsnittTekstboks ? (
+                            erLesevisning ? (
+                                skjema.felter.fritekstAvsnitt.verdi
+                            ) : (
+                                <Fieldset
+                                    legend="Legg til fritekst avsnitt"
+                                    hideLegend
+                                    id={fritekstSkjemaGruppeId}
+                                >
+                                    <StyledFamilieFritekstFelt>
+                                        <TextareaBegrunnelseFritekst
+                                            label={``}
```

Comment:
Vi må ha en label men du kan skjule den med propen `hideLabel`
```suggestion
                                            label="Skriv inn fritekstavsnitt"
                                            hideLabel
```